lib/puppet_litmus/serverspec.rb

```diff
@@ -22,6 +22,9 @@ def idempotent_apply(manifest)
   #  :catch_changes [Boolean] exit status of 1 if there were changes.  
   #  :expect_failures [Boolean] doesnt return an exit code of non-zero if the apply failed.  
   #  :manifest_file_location [Path] The place on the target system.
+  #  :prefix_command [String] prefixes the puppet apply command; eg "export LANGUAGE='ja';".
```

Comment:
The semi colon is not needed:
```suggestion
  #  :prefix_command [String] prefixes the puppet apply command; eg "export LANGUAGE='ja'".
```